.github/workflows/CI.yml

```diff
@@ -65,7 +65,6 @@ jobs:
           --continue
           jar
           validatePlugins
-          detekt
```

Comment:
check logs there are many violations!

Reply:
https://github.com/TWiStErRob/net.twisterrob.gradle/pull/457 was cancelled at exactly the right time to not get code scanning warnings.

Reply:
https://github.com/TWiStErRob/net.twisterrob.gradle/pull/454 was also merged with one new

Reply:
Branch protection rule was missing "detekt" as required.